kicad_AmigaAdapter/VideoSlot/V2/README.md

```diff
@@ -0,0 +1,42 @@
+# Amiga Video Slot Adapter V2
```

Comment:
Some additional thoughts that I think are worth covering before completing this as this PR will establish precedent.

1. There are a LOT of other boards out there now, some even more radical than this one. Many of those will just remain as forks of the project and never merge into main. That should be totally fine. So it is a good question of what should be here versus a fork.

2. What is version "N +1" is also a good question, versus just making another folder for an "alternate" build. The answer cannot be "whoever made the last radical redesign is the latest".

3. I like the idea that main contains a build that is easy to do, has only one hardware setting (which Denise), and can provide great picture quality for any Amiga it is put in. Right now the V1 is that if you add the 47pf capacitor (doesn't hurt and most likely helps). You might need to also tweak software, but not in all cases. So with that single change, we have an easy to build and configure board that should work for most if not all Amigas.

4. This board has a LOT of changes to it. Some seem arbitrary, whereas other seem warranted. It is hard to know what impact any single change has, for example the fill zones or the chip re-arrangement. What do we do with the next PR to come along that re-arranges them "just because"?

5. On the fill zones, I personally think this just makes future expansion more difficult, e.g. adding either more Pi hardware to the board, or making use of the other video slot signals (e.g. PPD, PBUSY, PPOUT, PSEL, etc.). There are some interesting possibilities there that if I were starting on, the V1 board is just simpler to update. Given what can be achieved with the V1 board in terms of picture quality, they don't seem required.

6. As of the moment this board has very little testing by others. I've got one on order and will be happy to share my experience. But is it ready to go in and look like it is a replacement for V1? I'm not sure yet, especially given the above and thoughts on future expansion. Perhaps at this time either a fork or an "alternate" sub-folder is better?

Reply:
Given your above concerns I'll remove this PR and keep a fork instead.

Reply:
To answer your questions though.. 1, 2 and 4 would typically be the job of project maintainers to sort out and give guidance on how such things should work.  3 fair enough, that is why I added a space for the 47pF. 5 KiCad deals with that automatically. 6 it has gone through extensive testing, but how much testing qualifies enough is another question entirely.